### PR TITLE
Update Maintainers.md

### PR DESCRIPTION
**Pull request by neewy**
### Description of the Change

Update maintainers information

### Benefits

Relevant information

### Possible Drawbacks 

This does not include information about inactive contributors from the past

### CODE REVIEW

File: MAINTAINERS.md [hunk 1]
```diff
@@ -1,21 +1,29 @@
...
-| Nikolay Yushkevich | neewy | nikolai@soramitsu.co.jp |
+Maintainers of Hyperledger Iroha project
+are supposed to help contributors by explain them project details,
+such as architecture, process, existing issues.
+
+This is the list of maintainers, including their email address for direct communications:
+
+|          Name          |     GitHub Id            |               email            |         Area of expertise       |
+|------------------------|--------------------------|--------------------------------|---------------------------------|
+| Makoto Takemiya        | takemiyamakoto           | takemiya@soramitsu.co.jp       | Product vision                  |
```
Comment (l4l) [inline MAINTAINERS.md]: maybe prepend `@` to the github id-s, so it will be clickable?

Comment (neewy) [reply MAINTAINERS.md]: @l4l 

Am I doing something wrong?

screen shot 2018-04-12 at 14 10 30

Comment (neewy) [reply MAINTAINERS.md]: It is not clickable

Comment (l4l) [reply MAINTAINERS.md]: :(

**Comment by neewy** [comment]
#1201 is fixed

**Pull request merged**